internal/serve/httphandler/receiver_handler.go

```diff
@@ -133,3 +133,12 @@ func (rh ReceiverHandler) GetReceivers(w http.ResponseWriter, r *http.Request) {
 
 	httpjson.RenderStatus(w, http.StatusOK, httpResponse, httpjson.JSON)
 }
+
+// GetReceiverVerification returns a list of verification types
+func (rh ReceiverHandler) GetReceiverVerificatioTypes(w http.ResponseWriter, r *http.Request) {
+	httpjson.Render(w, []data.VerificationField{
+		data.VerificationFieldDateOfBirth,
+		data.VerificationFieldNationalID,
+		data.VerificationFieldPin,
+	}, httpjson.JSON)
```

Comment:
Could you please move this to a function in the `data` package? Just like the [`GetAllRoles`](https://github.com/stellar/stellar-disbursement-platform-backend/blob/develop/internal/data/roles.go#L30-L37) method. This way, if a new `VerificationField` is added it automatically will be returned here.